week-2-wide-perspective/w02e05-simple-server-side/src/utils/jwt.ts

```diff
@@ -0,0 +1,29 @@
+import jwt from "jsonwebtoken";
+
+export interface Token {
+  valid: boolean;
+  expired: boolean;
+  decoded: {
+    userID: number;
+  };
+}
+export const createAccessToken = (id: number | undefined) => {
+  return jwt.sign({ userID: id }, "secret123", { expiresIn: "7d" });
+};
+
+export const verifyAccessToken = (token: string) => {
+  try {
+    const decoded = jwt.verify(token, "secret123");
```

Comment:
`secret123` powinien być jako zmienna `.env` ponieważ ktoś może dobrać się do kodu repo - odczytać to info i potencjalnie złamać Ci zabezpieczenia.